shared/common-adapters/standard-screen.js.flow

```diff
@@ -0,0 +1,11 @@
+/* @flow */
+import {Component} from 'react'
+
+export type Props = {
+children?: ?(React$Element<*> | Array<React$Element<*>>),
```

Comment:
Yes it should be indented.

I will look into seeing if there is a pre-made flow type, though my quick googling before did not reveal one.